lib/engine/game/g_1822/step/dividend.rb

```diff
@@ -92,8 +90,8 @@ def log_run_payout(entity, kind, revenue, subsidy, _action, payout)
             @log << "#{entity.name} earns subsidy of #{@game.format_currency(subsidy)}" if subsidy.positive?
```

Comment:
Subsidy is a parameter in this method but has no effect more than appearing in the log message if positive.

Reply:
That's correct. The purpose of the method is to log and there is a log for the subsidy if present.